packages/bot-web-ui/src/components/dashboard/dashboard-components/index.scss

```
@@ -0,0 +1,173 @@
+.bot-dashboard {
+    .toolbar__section {
+        justify-content: end;
+    }
+    .run-panel__container {
+        height: calc(100vh - 5rem) !important;
+        position: unset !important;
+        transform: translateX(calc(350px));
+        transition: all 0.4s;
+        margin-top: 1rem;
+    }
+    .dc-drawer--open {
+        transform: translateX(calc(0px)) !important;
```

Just 0 should do here.

```suggestion
        transform: translateX(calc(0)) !important;
```